Library/ViewModels/SettingsPrivacyViewModel.swift

```diff
@@ -81,7 +81,10 @@ SettingsPrivacyViewModelInputs, SettingsPrivacyViewModelOutputs {
    self.updateCurrentUser = Signal.merge(updatedFetchedUser,
                                          previousUserOnError)
 
-   self.refreshFollowingSection = self.didCancelSocialOptOutProperty.signal
+    self.refreshFollowingSection = Signal.merge(
```

Comment:
Both `Cancel` and `Confirm` now triggers `Following` section refresh in order for the VoiceOver to re-focus on the cell